soname bump for various libnm-glib and libnm-util API additions, like
IPoIB, bonding, and VLAN.

diff --git a/libnm-glib/Makefile.am b/libnm-glib/Makefile.am
index d11428b..21ba39c 100644
--- a/libnm-glib/Makefile.am
+++ b/libnm-glib/Makefile.am
@@ -138,7 +138,7 @@ libnm_glib_la_LIBADD = \
 	$(GUDEV_LIBS)
 
 libnm_glib_la_LDFLAGS = -Wl,--version-script=$(srcdir)/libnm-glib.ver \
-	-version-info "6:1:4"
+	-version-info "7:0:5"
 
 noinst_PROGRAMS = libnm-glib-test
 
diff --git a/libnm-util/Makefile.am b/libnm-util/Makefile.am
index c611261..30d75c7 100644
--- a/libnm-util/Makefile.am
+++ b/libnm-util/Makefile.am
@@ -63,7 +63,7 @@ libnm_util_la_SOURCES=			\
 libnm_util_la_LIBADD = $(GLIB_LIBS) $(DBUS_LIBS) $(UUID_LIBS)
 
 libnm_util_la_LDFLAGS = -Wl,--version-script=$(srcdir)/libnm-util.ver \
-	-version-info "7:0:6"
+	-version-info "8:0:7"
 
 if WITH_GNUTLS
 libnm_util_la_SOURCES += crypto_gnutls.c
